proca/test/proca_web/api/permissions_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule ProcaWeb.PermissionsTest do
+  @moduledoc """
+  This test checks only if the user has the permission to perform an action,
+  it does not check if the action actually succeeds.
+  """
+
+  use ProcaWeb.ConnCase
+  import Proca.StoryFactory, only: [red_story: 0]
+  import Proca.Factory
+
+  alias Proca.QueryFixtures
+
+  setup do
+    red_story()
+  end
+
+  @scenarios_map %{
+    add_campaign: %{
+      owner: :succeed,
+      campaigner: :succeed,
+      translator: :fail
+    },
+    change_campaign: %{
+      owner: :succeed,
+      campaigner: :succeed,
+      translator: :succeed
+    },
+    invite_owner: %{
```

Comment:
I'm not sure I get it. so the role of the invitee can be set by the inviter and they can't invite to higher level than themselves?

Reply:
Yes, exactly. Inviter can only invite people with the same or lower level of permissions